doc/rvv-intrinsic-spec.adoc

```diff
@@ -96,29 +98,33 @@ enum __RISCV_VXRM {
 
 NOTE: The RISC-V psABI ^9^ states that `vxrm` is not preserved across calls. Optimization for reducing the number of redundant writes to `vxrm` is a compiler and system specific issue.
 
-[[chapter-control-to-frm]]
+NOTE: Control to the vector fixed-point saturation flag (`vxsat`) ^22^ is not yet covered in the RVV C intrinsics v1.0. We plan to support them in follow-up versions in a compatible way with existing intrinsics in v1.0.
+
+[[control-of-frm]]
 === Control of `frm`
 
-For the floating-point intrinsics, representing the instructions under v-spec section 13, the intrinsics have two classes, called implicit-`frm` and explicit-`frm`.
+For the floating-point intrinsics, representing the floating-point arithmetic instructions ^23^, the intrinsics have two variants, called the implicit-`frm` and the explicit-`frm` intrinsics.
+
+NOTE: Control to the floating-point accrued exceptions flag fields (`fflag`) ^10^ is not yet covered in the RVV C intrinsics v1.0. We plan to support them in follow-up versions in a compatible way with existing intrinsics in v1.0.
 
 ==== Implicit-`frm` intrinsics
 
 The implicit-`frm` intrinsics behave like any C-language floating-point expressions, using the default rounding mode when `FENV_ACCESS` is off, and using the `fenv` dynamic rounding mode when `FENV_ACCESS` is on.
 
 NOTE: Both GNU and LLVM compilers generate scalar floating-point instructions using dynamic rounding mode, relying on the kernel initialization to set `frm` to `RNE` (specified as "roundTiesToEven" in IEEE-754 (a.k.a. IEC 60559)).
 
-NOTE: The implicit-`frm` intrinsics are intended to be used regardless of `FENV_ACCESS`. They are provided when `FENV_ACCESS` is on for the (few) programmers who are already using fenv. And they are provided when `FENV_ACCESS` is off for the (vast majority of) programmers who prefer the default rounding mode.
+NOTE: The implicit-`frm` intrinsics are intended to be used regardless of `FENV_ACCESS`. They are provided when `FENV_ACCESS` is on for the (few) programmers who are already using fenv; and they are provided when `FENV_ACCESS` is off for the (vast majority of) programmers who prefer the default rounding mode.
 
-[[chapter-explicit-frm]]
+[[explicit-frm]]
 ==== Explicit-`frm` intrinsics
 
 The explicit-`frm` intrinsics contain the `frm` operand which indicates the rounding mode (`frm`) ^10^ control. The floating-point intrinsics with the `frm` operand are followed by an `_rm` suffix in the function name.
 
-The `frm` operand is required to be an integer constant expression. The compiler should implement the following enum that maps to the defined rounding mode values under RISC-V ISA Manual Table 8.1 ^8^.
+The `frm` operand is required to be an integer constant expression. The compiler should implement the following enum that maps to the defined rounding mode values under RISC-V ISA Manual Table 8.1 ^9^.
```

Comment:
there was a "integer constant expression" → "constant integer expression" change in another place, should it be here too?